src/pages/Facility/settings/organizations/FacilityOrganizationIndex.tsx

```diff
@@ -80,38 +93,235 @@ export default function FacilityOrganizationIndex({
+                  size={"icon"}
+                  variant={"link"}
+                  onClick={() => toggleRow(org.id)}
+                  className="px-0"
+                >
+                  {expandedRows[org.id] ? (
+                    <CareIcon icon="l-angle-down" className="h-5 w-5" />
+                  ) : (
+                    <CareIcon icon="l-angle-right" className="h-5 w-5" />
+                  )}
+                </Button>
+              )}
+              {org.name}
+            </div>
+            {isTopLevel && (
+              <div className="flex gap-5 justify-between">
+                {children.length > 0 && (
+                  <div>
+                    <TooltipProvider>
+                      <Tooltip>
+                        <TooltipTrigger>
+                          <Button
+                            variant="white"
+                            size={isMobile ? "xs" : "sm"}
+                            onClick={toggleAllChildren}
+                          >
+                            <CareIcon
+                              icon={allExpanded ? "l-minus" : "l-plus"}
+                              className="h-4 w-4 sm:h-2 sm:w-2"
+                            />
+                            <span className="hidden sm:inline">
+                              {t(allExpanded ? "collapse_all" : "expand_all")}
+                            </span>
+                          </Button>
+                        </TooltipTrigger>
+                        <TooltipContent>
+                          {t(allExpanded ? "collapse_all" : "expand_all")}
+                        </TooltipContent>
+                      </Tooltip>
+                    </TooltipProvider>
+                  </div>
+                )}
+                <div className="ml-auto">
+                  <TooltipProvider>
+                    <Tooltip>
+                      <TooltipTrigger>
+                        <Button
+                          variant="white"
+                          size={isMobile ? "xs" : "sm"}
+                          asChild
+                        >
+                          <Link
+                            href={`/departments/${org.id}`}
+                            className="text-gray-900 flex items-center"
+                          >
+                            <CareIcon icon="l-eye" className="h-4 w-4" />
+                            <span className="hidden sm:inline">
+                              {t("see_details")}
+                            </span>
+                          </Link>
+                        </Button>
+                      </TooltipTrigger>
+                      <TooltipContent>{t("see_details")}</TooltipContent>
+                    </Tooltip>
+                  </TooltipProvider>
+                </div>
+              </div>
+            )}
+          </TableCell>
+          <TableCell className="border-l bg-white font-semibold text-gray-900">
+            {org.org_type}
```

Comment:
```suggestion
            t(`facility_organization_type__${org.org_type}`)}
```

@DonXavierdev  Let's use translations

Reply:
@abhimanyurajeesh I feel this is unnecessary since we already have a header that says "Category". This would be repetitive in every row under it.

Reply:
we'll still need translations anyhow to map it, be it expanded or short form

Reply:
@rithviknishad So should there be like dynamic translations for every category names and department names ?